### PR TITLE
Add _.round/_.floor/_.ceil

### LINKED ISSUE #1001: _.round
@jdalton you [knew this was coming](https://github.com/lodash/lodash/pull/987#issuecomment-75475069). :stuck_out_tongue_winking_eye: 

It would be super useful to have a `Math.round` equivalent that supported the precision that it can be rounded to. 

```js
_.round(4.006);
// => 4

_.round(4.006, 2);
// => 4.01

_.round(4060, -2);
// => 4100
```

It also would be ultra simple to add support for [`_.ceil` and `_.floor`](https://github.com/lodash/lodash/pull/987#issuecomment-75475528) without [duplicating code](http://jsbin.com/jacusa/1/edit?js,console).

### PR DESCRIPTION
**Pull request by thejameskyle**
Resolves #1001

### CODE REVIEW

**Comment by jdalton** [comment]
:ok_hand:

**Comment by thejameskyle** [comment]
:dancer:

File: lodash.src.js [hunk 1]
```diff
@@ -3707,6 +3707,25 @@
...
+     *
+     * @private
+     * @param {String} type Specify the Math method to use when rounding.
+     */
+
+    function createRound(type) {
+      var roundFn = Math[type];
+      return function(number, precision) {
+        if (precision) {
+          precision = Math.pow(10, precision);
```
Comment (megawac) [inline lodash.src.js]: worth caching these?

Comment (jdalton) [inline lodash.src.js]: You [know how I do](https://github.com/lodash/lodash/commit/c8b818f011257b09f98d12fabde92ba5a77a43d3#diff-a0ac9c4b57e4a35674426f2b910a39daR754).

**Pull request merged**